### PR TITLE
feat: improve devX for typescript sdk sample

### PR DESCRIPTION
**Pull request by grant**
This change runs through the Runtime SDK TypeScript sample and makes a few improvements.

- Use consistent `NPM_TOKEN` instead of `ANON_NPM_TOKEN` as seen in 1Password
- Describe what this sample does (connects to IG)
- Instruct the user how to setup the env vars
- Instruct the user how to setup the `.npmrc` file
- Log errors for missing env vars
- Load env vars properly from the correct directory

### CODE REVIEW

**Comment by grant** [comment]
Will address comments and rebase this later today.

File: runtime-sdk-typescript/README.md [hunk 1]
```diff
@@ -1,29 +1,55 @@
...
+NPM_TOKEN=...
+```
+
+### NPM Token
+
+Set your `NPM_TOKEN` environment variable in the file `.npmrc` using this command:
 
 ```sh
-sed "s/\${ANON_NPM_TOKEN}/${ANON_NPM_TOKEN}/g" .npmrc.template >.npmrc
+source ../.env
```
[the remark this answers is not in the document]
Comment (grant) [reply runtime-sdk-typescript/README.md]: We don't need it, we can remove it thanks!

File: runtime-sdk-typescript/README.md [hunk 2]
```diff
@@ -1,29 +1,55 @@
...
 ```sh
-# If you're using npm
 npm install
-npx playwright install
-
-# If you're using yarn
-yarn install --update-checksums
-yarn add @playwright/test
+# or
+yarn
```
[the remark this answers is not in the document]
Comment (grant) [reply runtime-sdk-typescript/README.md]: - We should add playwright to dependencies if it is a dependency to run the application
- Why would we need to update the lockfile with [`--update-checksums`](https://classic.yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-update-checksums) when running this app as a developer?

**Comment by grant** [comment]
Addressed comments, PTAL @jeffreyyun ! :)

**Review by jeffreyyun — APPROVED** [review]
tested locally as well, great!!!

**Pull request merged**